app/templates/error.hbs

```diff
@@ -0,0 +1,10 @@
+<div data-test-error-page class="editor-page">
+  <div class="container page">
+    <div class="row">
+      <div class="col-md-10 offset-md-1 col-xs-12">
+        <h2>Sorry but that page does not exist.</h2>
+        <p>Try find your page from the {{#link-to "home"}}homepage{{/link-to}}</p>
```

Comment:
Maybe should be:
"Try `to` find your page from the homepage"

Reply:
It reads fine to me 🙈 but then again I'm not sure if your suggestion is grammatically correct? 😄

Reply:
I am not sure if "Try find" makes sense 😅